homeassistant/components/sensor/darksky.py

```diff
@@ -97,6 +97,346 @@
                              ['currently', 'hourly', 'daily']],
 }
 
+CONDITION_PICTURES = {
+    'clear-day': ('data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZ'
```

Comment:
Placing specific weather icons in the images folder seems a little out of place. I looked more at the code, and I think I found the relevant changes that need to be made to the frontend component in order to support weather icons. I'm gonna do that, and once it's done - every weather sensor will be able to display standardizes Material Design icons.